README.md

```diff
@@ -218,11 +218,21 @@ brew install curl cmake libressl c-ares
 
 #### Windows (using vcpkg)
 
+Probably the easiest way to download and compile `google-cloud-cpp` on Windows
+is to use [vcpkg][vcpkg-link], a package manager from Microsoft that downloads
+the source code for a library (and all its dependencies), compiles it, and adds
+the right configuration for DevStudio and/or CMake.
+
+After downloading and compiling `vcpkg`, simply run:
+
 ```bash
-set PROVIDER=vcpkg
-set GENERATOR="Visual Studio 15 2017 Win64"
-powershell -exec bypass .\ci\install-windows.ps1
+.\vcpkg.exe install google-cloud-cpp:x64-windows-static
 ```
+
+and then follow the instructions from `vcpkg`. We try to keep the version of
+`google-cloud-cpp` up-to-date, our practice is to submit a PR to update the
+version in `vcpkg` after each release of `google-cloud-cpp`.
```

Comment:
Is the information on updating the version necessary for a Windows developer?  I don't know much about vcpkg but it seems out of scope.

Reply:
We had a customer hesitating to use `vcpkg` because they thought it might be too old.  I want to give folks a sense that this is kept reasonably up to date.

Reply:
Sg, thanks for clarifying.